website/language.py

```diff
@@ -72,6 +72,21 @@
 
 INVALID_EMAIL_CONFIRM_TOKEN = 'This confirmation link is invalid. Please <a href="/login/">log in</a> to continue.'
 
+CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
+
+CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a conflict.'
```

Comment:
Yes, that's what the user sees. Listing the conflicts preventing the merge would be reasonable, but I didn't want to go down that path until we had some feedback on what it should do.

My preference would be to push to staging the way it is, and follow up with an improvement that meets the observed needs of QA based on their feedback before publication.